source/includes/language-compatibility-table-go.rst

```diff
@@ -10,8 +10,10 @@ The first column lists the driver version.
 
    * - Go Driver Version
      - Minimum Go Language Version
+   * - 1.11
+     - 1.13 
    * - 1.10
-     - 1.13
+     - 1.10
```

Comment:
According to [DOCSP-26304](https://jira.mongodb.org/browse/DOCSP-26304) this was incorrectly documented before.